open-craft/accounting
# Use Pipfile and pipenv.

This PR uses the new pipfile/pipenv scheme of dependency and virtual environment management.

**Testing Instructions**:

* Ensure this works: `make dev.down` -> `make dev.build` -> `make dev.up` -> `make dev.web.logs` & `make shell` & `curl http://localhost:1786/admin`.
* Ensure this works: `make dev.down` -> `vagrant up --provision` -> `vagrant ssh` -> `make run.dev` & `make shell` & `curl http://localhost:1786/admin`.

The complementary ops PR for this is at https://github.com/open-craft/ansible-accounting/pull/2.
